### ISSUE TITLE
Error Using pip install dslogparser

### ISSUE THREAD

**Issue by warrenlp** [ligerbots/dslog2csv#4]
I seem to be able to import other packages from pip and I even attempted using the straight .tar.gz file for this but I am getting the following error:

_(base) C:\Users\warre\Anaconda3\Scripts>pip install dslogparser
Collecting dslogparser
  Using cached dslogparser-1.0.2.tar.gz (6.2 kB)
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\warre\anaconda3\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\warre\\AppData\\Local\\Temp\\pip-install-is7sibmd\\dslogparser\\setup.py'"'"'; __file__='"'"'C:\\Users\\warre\\AppData\\Local\\Temp\\pip-install-is7sibmd\\dslogparser\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\warre\AppData\Local\Temp\pip-install-is7sibmd\dslogparser\pip-egg-info'
         cwd: C:\Users\warre\AppData\Local\Temp\pip-install-is7sibmd\dslogparser\
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\warre\AppData\Local\Temp\pip-install-is7sibmd\dslogparser\setup.py", line 43, in <module>
        with open(join(setup_dir, 'README-dslogparser.md'), encoding='utf-8') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\warre\\AppData\\Local\\Temp\\pip-install-is7sibmd\\dslogparser\\README-dslogparser.md'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output._

Is there a setup tool issue here?

Update:
I was able to clone this code and create my own wheel:
python setup.py bdist_wheel
cd dist
pip install *whl

**Comment by jaustinpage** [comment]
i will try to take a look at this tomorrow.

**Comment by prensing** [comment]
I got the same problem when I uninstalled and then reinstalled. Looking...

**Comment by prensing** [comment]
I think this is fixed with version 1.0.3 (just uploaded).

Justin: I created 2 README files because "README.md" is displayed by Github and holds both a description and historic notes. However, using that file for the PyPi package description seemed wrong, too wordy. So I created "README-dslogparser.md" to provide a more concise and readable PyPi description, but missed loading it into the package. If you have a different suggestion on how to organize this, I am happy to hear.

**Comment by jaustinpage** [comment]
nah, dont want to complicate it. it looks good to me :-)
